### PR TITLE
APM-363 Document payment embeds

### PR DESCRIPTION
**Pull request by willemstuursma**

### CODE REVIEW

File: source/reference/v2/captures-api/get-capture.rst [hunk 1]
```diff
@@ -38,6 +38,13 @@ If you are using :doc:`organization access tokens </guides/authentication>` or a
 
      - Set this to ``true`` to retrieve a test mode capture.
 
+Embedding of related resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This endpoint allows for embedding additional information by appending the following values via the ``embed``
+query string parameter.
+
+* ``payment`` Include the :doc:`payment </reference/v2/payments-api/get-payment>` the capture were created for.
```
Comment (ndijkstra) [inline source/reference/v2/captures-api/get-capture.rst]:
```suggestion
* ``payment`` Include the :doc:`payment </reference/v2/payments-api/get-payment>` where the capture was created for.
```

File: source/reference/v2/captures-api/list-captures.rst [hunk 2]
```diff
@@ -22,6 +22,13 @@ Parameters
 ----------
 Replace ``paymentId`` in the endpoint URL by the payment's ID. For example: ``/v2/payments/tr_7UhSN1zuXS/captures``.
 
+Embedding of related resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This endpoint allows for embedding additional information by appending the following values via the ``embed``
+query string parameter.
+
+* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` the captures were created for.
```
Comment (ndijkstra) [inline source/reference/v2/captures-api/list-captures.rst]:
```suggestion
* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` where the captures were created for.
```

File: source/reference/v2/settlements-api/list-settlement-captures.rst [hunk 3]
```diff
@@ -16,15 +16,21 @@ List settlement captures
...
 ----------
 Replace ``settlementId`` in the endpoint URL by the settlement's ID. For example:
 ``/v2/settlements/stl_jDk30akdN/captures``.
 
+Embedding of related resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This endpoint allows for embedding additional information by appending the following values via the ``embed``
+query string parameter.
+
+* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` the captures were created for.
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-captures.rst]:
```suggestion
* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` where the captures were created for.
```

File: source/reference/v2/settlements-api/list-settlement-chargebacks.rst [hunk 4]
```diff
@@ -21,6 +21,13 @@ Replace ``settlementId`` in the endpoint URL by the settlement's ID, for example
 This endpoint is an alias of the :doc:`List chargebacks </reference/v2/chargebacks-api/list-chargebacks>` endpoint. All
 parameters for that endpoint can be used here as well.
 
+Embedding of related resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This endpoint allows for embedding additional information by appending the following values via the ``embed``
+query string parameter.
+
+* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` these chargebacks were issued for.
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-chargebacks.rst]:
```suggestion
* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` where the chargebacks were issued for.
```

File: source/reference/v2/settlements-api/list-settlement-refunds.rst [hunk 5]
```diff
@@ -21,6 +21,13 @@ Replace ``settlementId`` in the endpoint URL by the settlement's ID, for example
 This endpoint is an alias of the :doc:`List refunds </reference/v2/refunds-api/list-refunds>` endpoint. All parameters
 for that endpoint can be used here as well.
 
+Embedding of related resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This endpoint allows for embedding additional information by appending the following values via the ``embed``
+query string parameter.
+
+* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` the refunds were created for.
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-refunds.rst]:
```suggestion
* ``payment`` Include the :doc:`payments </reference/v2/payments-api/get-payment>` where the refunds were created for.
```

Comment (willemstuursma) [reply source/reference/v2/settlements-api/list-settlement-refunds.rst]: I think adding "where" is a Dutchism.

File: source/reference/v2/captures-api/get-capture.rst (hunk 1 above)
Comment (willemstuursma) [reply source/reference/v2/captures-api/get-capture.rst]: One payment is embedded per refund, if multiple payments were embedded it should be `payments`. Which one has `payments`?

**Pull request merged**